New package: PhysicalFFT v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: PhysicalFFT
- Repository: https://github.com/JuliaAstroSim/PhysicalFFT.jl
- Created by: @islent
- Version: v0.1.0
- Commit: dd3b90ec6ed886cf41ecf6c0739e38d57071996b
- Git reference: HEAD
- Description: FFT solvers in Julia language

goerz commented:
Can you add a paragraph to the README to explain how this FFT does or does not differ from "standard" FFT? Why is it "Physical"?

islent commented:
Thanks for pointing out the problem. I've added these info in Readme.

This package currently support solving gravitational Poisson equation using "standard" Fourier transform method.

We intentionally take the name of it similar to `PhysicalFDM`, `PhysicalTrees` and `PhysicalMeshes`, which are gravitational (physical) Poisson solvers integrated in package `AstroNbodySim`.

Another reason of using `Physical` is to avoid conflicts of `Unitful` packages, nevertheless we support unitful types in all these packages.

[noblock]

JuliaHubRegistrator committed:
New package: PhysicalFFT v0.1.0

UUID: d41fe92e-7e57-414e-90fb-fae81d34872f
Repo: https://github.com/JuliaAstroSim/PhysicalFFT.jl.git
Tree: 61fe0c64f55d8e23625da6d4ab37a5e3af56a9e4

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
